brambling/models.py

```diff
@@ -1361,3 +1370,129 @@ def send(self, site, body_template_name='brambling/mail/invite_{kind}_body.html'
         )
         self.is_sent = True
         self.save()
+
+
+class CustomForm(models.Model):
+    ATTENDEE = 'attendee'
+    ORDER = 'order'
+
+    FORM_TYPE_CHOICES = (
+        (ATTENDEE, _('Attendee')),
+        (ORDER, _('Order')),
+
+    )
+    form_type = models.CharField(max_length=8, choices=FORM_TYPE_CHOICES,
+                                 help_text='Order forms will only display if "collect survey data" is checked in your event settings')
+    event = models.ForeignKey(Event, related_name="forms")
+    # TODO: Add fk/m2m to BoughtItem to limit people the form is
+    # displayed to.
+    name = models.CharField(max_length=50)
```

Comment:
If people have multiple forms for Order and Attendee, we might want to display the name someday.